puppet/ha-puppet/screenshot.js

```diff
@@ -282,7 +356,7 @@ export class Browser {
           x: 0,
           y: HEADER_HEIGHT,
           width: viewport.width,
-          height: viewport.height - HEADER_HEIGHT,
+          height: viewport.height,
```

Comment:
this is wrong. The height has to be offset with `HEADER_HEIGHT` because the y is set to start at `HEADER_HEIGHT`

Reply:
I would expect to get a 1000x1000px image in return when using `viewport=1000x1000` (?)

Reply:
Yep, and we increased the viewport height by 56px earlier.